server/questionsAPI.js

```diff
@@ -0,0 +1,22 @@
+require('dotenv').config();
+const axios = require('axios');
+
+const baseURL = 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe';
+
+const getQuestions = (params, callback) => {
+  const options = {
+    method: 'GET',
+    url: `${baseURL}qa/questions?product_id=${params}&count=20`,
+    headers: { Authorization: process.env.GITHUBKEY },
```

Comment:
make authorization header a constant at the top (like baseURL)
`const headers = {
  Authorization: process.env.GITHUBKEY,
};`